src/app/containers/ATIAnalytics/params/mostReadPage/buildParams.js

```diff
@@ -0,0 +1,43 @@
+import { buildATIPageTrackPath } from '../../atiUrl';
+import { LIBRARY_VERSION } from '#lib/analyticsUtils';
+
+export const buildMostReadATIParams = (
+  pageData,
+  requestContext,
+  serviceContext,
+) => {
+  const { platform, statsDestination } = requestContext;
+  const {
+    atiAnalyticsAppName,
+    atiAnalyticsProducerId,
+    brandName,
+    lang,
+    service,
+    mostRead: { header },
+  } = serviceContext;
+
+  return {
+    appName: atiAnalyticsAppName,
+    contentType: 'list-datadriven',
+    language: lang,
+    pageIdentifier: `${service}.popular.read.page`,
+    pageTitle: `${header} - ${brandName}`,
+    producerId: atiAnalyticsProducerId,
+    libraryVersion: LIBRARY_VERSION,
+    statsDestination,
+    platform,
+    service,
+  };
+};
```

Comment:
Wondering if it might worth logging the `timePublished` and `timeUpdated` as well.

Reply:
I left it out because it's provided in the current most read page doesn't pass. But with further inspection in the data we have available for most read. I will pass in these values from the data response, https://www.bbc.com/pidgin/mostread.json
```
"lastRecordTimeStamp"
"firstRecordTimeStamp"
```